src/V3Const.cpp

```diff
@@ -885,6 +885,8 @@ class ConstVisitor final : public VNVisitor {
     bool m_doV = false;  // Verilog, not C++ conversion
     bool m_doGenerate = false;  // Postpone width checking inside generate
     bool m_hasJumpDelay = false;  // JumpGo or Delay under this while
+    bool m_resolvedNets
+        = false;  // Enable using values of nets, multiple drivers are resolved in V3Tristate
```

Comment:
This is very minor, but either break up the comment into two lines or put it above the member so that the init expression is on the same line as the member.

Reply:
It was done by clang-format

Reply:
But I can break a comment by myself and it won't be rejoined.